.github/workflows/lint.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-node@v3
         with:
-          node-version: 10
+          node-version: 12
```

Comment:
I still think we should use v16 here too. Make sure we are using the latest LTS release like in the release workflow.